src/app/shared/constants/constants.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export const CIDR_ALLOW_ALL = '0.0.0.0/0';
```

Comment:
Export this to a static class similar to how it's done in `src/app/shared/validators/validators.ts`.

Reply:
Why was it marked as is resolved without any change or comment?

Reply:
I fixed this locally and marked it as resolved since I was about to push my changes. But then found out a few other issues :)

Reply:
Logic has been modified for this, PTAL.